src/common.cpp

```diff
@@ -124,8 +291,141 @@ namespace dynamicgraph
 	  q_urdf[idx]=q_sot[m_urdf_to_sot[idx]];	
 	return true;
       }
+      
+      bool RobotUtil::
+      velocity_urdf_to_sot(Eigen::ConstRefVector v_urdf, Eigen::RefVector v_sot)
+      {
+	assert(q_urdf.size()==m_nbJoints+6);
```

Comment:
Just for your information this pull request is closed.
I rebased the code on the latest commit from proyan and this is fixed in
https://github.com/stack-of-tasks/sot-torque-control/pull/22